roseus_smach/src/state-machine-ros.l

```diff
@@ -14,7 +14,7 @@
 (defclass state-machine-inspector
   :super propertied-object
   :slots (sm root-name srv-name init-tm state-counter structure-counter
-             groupname))
+             groupname exit-state))
```

Comment:
Why do we need to add this slot again?
Is it only to avoid double-calling `:register-exit-signal-hook`?
Is that double-calling prevention a feature that we really want?

Reply:
we return `exit-state` when `:register-exit-signal-hook` with no args.

Reply:
Oh, I see. I would still prefer to have a separate method (like `:exit-state`) for this, since the naming `:register-exit-signal-hook` seems a little too specific for this behavior.

If we are introducing the slot I would also like to see it actually used in the callback below. Otherwise, I still don't think we need to introduce a new slot.

Reply:
I updated.